Adding training plan proposal (#142)
ome/omero_search_engine: docs/training_plan/training_pln.rst

Training Plan
=============

Admin Training
--------------
Objective: Teach administrators how to install, configure, and manage the IDR-searcher.

* IDR-searcher Layout
    * System requirements
    * Required software and dependencies
* Introduction to Ansible
    * Overview of Ansible
    * Understanding Ansible roles
* IDR-searcher Role Overview
    * Explanation of the application role
    * Review of the variables used in the role
* IDR-searcher Installation
    * How to configure the Ansible playbook
    * Running the playbook to install the application
* Administration Commands
    * Overview of admin commands

>     * How to use them to configure and run the IDR-searcher

Reviewer: ```suggestion
    * How to use admin commands to configure and run the IDR-searcher
```

* Data Indexing
    * How to perform data indexing
    * Generating and managing cache files
* Container File Creation
    * CSV format
    * Parquet format
    * SON format
* Validation and Testing
    * Testing the IDR-searcher installation
    * Verifying indexing and functionality
* Backup and Recovery
    * Backup procedures
    * Restore process

>     * disaster recovery basics

Reviewer: ```suggestion
    * Disaster recovery basics
```

User Training
-------------
Objective: Teach users how to effectively use the IDR-searcher and its APIs.

* IDR-searcher Introduction
    * Overview of the IDR-searcher and its purpose
* IDR-searcher Capabilities
    * Understanding the information and insights available through the IDR-searcher
* Swagger Documentation Overview
    * Introduction to the Swagger interface
    * Navigating the API documentation
* API Query Types
    * Understanding GET and POST requests

>     * When and how to use each type

Reviewer: ```suggestion

```

I would leave this out. imho, this is a part of``understanding``.

If need be though, just reformulate as 

``When and how to use GET an POST``
